README.md

```diff
@@ -404,9 +416,27 @@ button, input.c-button {
 }
 ```
 
-IMPORTANT: **Never use id's in  selectors**. They have very high specificity compared to classes, and can't be used more than once on the same page.
+<a name="keep-specificity-low"></a>
+#### Keep specificity low.
+
+* Type selectors (addressing elements by their tag name) should be defined early in the cascade and then never used again unless absolutely necessary.
+
+* Selectors consisting of a single class should make up the majority of your CSS. A second class should generally only be added for states. 
+
+* **NEVER use id's in selectors**. They have very high specificity compared to classes, and can't be used more than once on the same page.
+
+* For the same reason, **NEVER use inline styles**. They are near-impossible to override without resorting to JavaScript.
+
+* It should go without saying, **don't use `!important`** except in rare cases.
```

Comment:
Give a couple examples of exceptions to this rule. Utilities use this, but what is another "rare case"?